README.md

```diff
@@ -1,2 +1,77 @@
-# Module-Two-Capstone-Project
-Using Kanban board with filled-in tasks that have categories and time estimations to assign work in group.
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# TVMAZE : Movie website:
+![TVMAZE-logo](./src/images/tvm_api.png)
+> This JavaScript capstone project is built on the [TVMAZE API](https://www.tvmaze.com/api), an external API, and Microverse's in-house [Involvement API](https://www.notion.so/microverse/Involvement-API-869e60b5ad104603aa6db59e08150270). The cards display the movies information that was retrieved from the API. The user has the option to like and comment on the sepecific movie.
+
+
+## Built With
+
+- HTML5, JavaScript, and API
+- SASS
+- JEST TEST
+- Linters:
+  - Stylelint, Lighthouse, Webhint and ESLint
+
+## Project Documentation
+
+- Here is the [link]() to our Zoom video presentation
+
```

Comment:
- [ ] Please, consider that this link to your ZOOM video record is broken:
![image](https://user-images.githubusercontent.com/49064106/189501213-dc712f47-cd37-4d7d-857e-9efaa8d1a6f7.png)